src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -235,7 +277,8 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
                 );
                 importModelsList.push(newNode, ...relationships);
             });
-            setElements([...importModelsList]);
+            const layoutedElements = getLayoutedElements([...importModelsList]);
```

Comment:
`layoutedElements` / `getLayoutedElements` --> might be better as `positionedElements` / `applyLayoutToElements`? `layouted` isn't quite right 🤔